.vscode/launch.json

```diff
@@ -5,6 +5,10 @@
             "type": "cppvsdbg",
             "request": "launch",
             "program": "${config:cmake.installPrefix}/bin/audacity.exe",
+            "args": [
+                // Comment this out to run audacity in plugin registration mode
```

Comment:
You could add another configuration "run audacity (plugin registration)", just make a copy of the current one